internal/utils.py

```diff
@@ -1,16 +1,15 @@
 """stores utils that are shared between scripts"""
+import os
 import argparse
-import yaml
 
-with open("config.yml", "r") as ymlfile:
-    config = yaml.load(ymlfile, Loader=yaml.FullLoader)
+from internal.config import config
 
 
 def get_talents(args):
     """lookup talents based on current config"""
     if args.talents:
         talents = [args.talents]
-    elif config["sims"][args.dir[:-1]]["builds"]:
+    elif config["sims"][args.dir]["builds"]:
```

Comment:
wont this fail? Since `args.dir` is `talents/` but we need to lookup based on `talents`

Reply:
No since we now normalise the path, so a user could do `./talents` or `./talents/` it would be normalised to `talents`